src/sql/run.py

```diff
@@ -154,6 +154,23 @@ def _repr_html_(self):
         else:
             return None
 
+    def _repr_text(self):
+        if self.pretty:
+            self.pretty.add_rows(self)
+            result = str(self.pretty)
+            if self.truncated:
+                result += (
```

Comment:
Instead of using `+` we should use the string format

Reply:
Sure